spec/index.bs

```diff
@@ -2483,6 +2483,9 @@ enum GPUCompareFunction {
         - |descriptor|.{{GPUSamplerDescriptor/lodMinClamp}} is greater than or equal to 0.
         - |descriptor|.{{GPUSamplerDescriptor/lodMaxClamp}} is greater than or equal to
             |descriptor|.{{GPUSamplerDescriptor/lodMinClamp}}.
+        - When |descriptor|.{{GPUSamplerDescriptor/maxAnisotropy}} is greater than 1,
+            |descriptor|.{{GPUSamplerDescriptor/magFilter}}, |descriptor|.{{GPUSamplerDescriptor/minFilter}},
+            and |descriptor|.{{GPUSamplerDescriptor/mipmapFilter}} must be equal to {{GPUFilterMode/"linear"}}.
```

Comment:
Oops, when I looked at this earlier and suggested a change, I didn't realize we actually did already have spec text for validating GPUSamplerDescriptor. I recently added the "It must be &ge; 1" line in the wrong place; I should have added it here. Would you mind just moving your added validation to this algorithm and deleting "must be &ge; 1" line above?